myapp/app/models/task.rb

```diff
@@ -74,4 +77,16 @@ def tag_list=(names)
   def self.tagged_with(name)
     Tag.find_by(name: name)&.tasks || Task.none
   end
+
+  def editable_user_list
+    editable_users.map(&:email).join(', ')
+  end
+
+  def editable_user_list=(emails)
+    self.editable_users = emails.split(',').map do |email|
+      user_id = User.find_by!(email: email.strip).id
```

Comment:
I know it's not ideal if user input emails that no user has, but just to make it simple here as a better solution will involve more complicated FE works. 😅